src/DIRAC/WorkloadManagementSystem/Utilities/PilotCStoJSONSynchronizer.py

```diff
@@ -244,6 +244,29 @@ def _getPilotOptionsPerSetup(self, setup, pilotDict):
                     return queueOptionRes
                 queuesDict[queue] = queueOptionRes["Value"]
             pilotDict["Setups"][setup]["Logging"]["Queues"] = queuesDict
+        elif "loggingRESTService" in pilotDict["Setups"][setup]:
+            self.log.debug(
+                "Getting option of ", "/DIRAC/Setups/%s/%s" % (setup, pilotDict["Setups"][setup]["loggingRESTService"])
```

Comment:
Does not look like a suitable location.

Reply:
I'm wondering if we really need this (a section in pilot.json). It was there in the original code. Now we sending logs to Tornado, and in the second step are selecting a plugin by name (`TornadoPilotLoggingHandler` does this) from the CS directly (MQ or File cache) and we don't need this section. I would revert to the original code here.

Reply:
Indeed we might not need it.